styles/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$breakpoint-sm: 600px; //Mobile (small devices)
+$breakpoint-lg: 601px; //Large devices like desktop
```

Comment:
Regarding my earlier comment on why I prefer semantically named sizes for breakpoints, the fact that a comment is needed to indicated what size this refers to is, for me at least, an indication that the names should be more semantic.